modules/admin_manual/pages/configuration/files/manual_file_locking.adoc

```diff
@@ -9,7 +9,7 @@ The feature builds on the WebDAV Locks backend which has been introduced with Se
 
 == Enable or Disable the UI Component
 
-NOTE: The _user-facing_ components in the web interface are disabled by default because this feature allows users to lock other users' files *exclusively*. Even the owner of the file can't unlock them, only the locking user can unlock until the lock expires.
+NOTE: The _user-facing_ components in the web interface are disabled by default because this feature allows users to lock other users' files *exclusively*. Even the owner of the file can't unlock them. Lock releasing can only be done by the locking user, a user being member of a lock breaking group if defined, or until the lock expires.
```

Comment:
```suggestion
NOTE: The _user-facing_ components in the web interface are disabled by default because this feature allows users to lock other users' files *exclusively*. Even the owner of the file can't unlock them. Lock releasing can only be done by the locking user, a user who is a member of a lock breaking group (if defined), or until the lock expires.
```